### PR TITLE
[KERNEL32] Threadpooling API functions

### PR DESCRIPTION
**Pull request by Getequ**
taken from wine/dlls/kernel32/thread.c

this PR is possible to merge only after #1190 ! So this one always will be red without him

### CODE REVIEW

**Comment by Getequ** [comment]
> Must go in kernel32_vista ?

kernel32_vista  contains just 14 functions. May be better do an opposite and put under them -version=0x600 in spec?

File: dll/win32/kernel32/CMakeLists.txt [hunk 1]
```diff
@@ -112,7 +112,7 @@ set_subsystem(kernel32 console)
 ################# END  HACK #################
 
 target_link_libraries(kernel32 wine chkstk ${PSEH_LIB})
-add_importlibs(kernel32 ntdll)
+add_importlibs(kernel32 ntdll ntdll_vista)
```
Comment (ThFabba) [inline dll/win32/kernel32/CMakeLists.txt]: New imports go on the left, ntdll should always be last in the list.

File: dll/win32/kernel32/client/thread.c [hunk 2]
```diff
@@ -1251,4 +1251,194 @@ TlsSetValue(IN DWORD Index,
     return TRUE;
 }
 
+
```
Comment (ThFabba) [inline dll/win32/kernel32/client/thread.c]: I think putting this stuff in a (new) file under dll/win32/kernel32/wine might make more sense for easier syncing. Also please add a mention in README.WINE.

File: dll/win32/kernel32/CMakeLists.txt (hunk 1 above)
Comment (tkreuzer) [inline dll/win32/kernel32/CMakeLists.txt]:
```suggestion
add_importlibs(kernel32 ntdll_vista ntdll)
```

**Pull request closed without merging**